### PR TITLE
React layout

### PR DESCRIPTION
**Pull request by aminalakhsham**
Relates #48

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,12 +1,56 @@
...
+  };
+  render() {
+    const { login } = this.state;
+    return (
+      <BrowserRouter>
+        {login ? (
+          <>
+            <Header />
+            <Switch>
+              <Route exact path={"/"} Redirect={"/projects"} />
```
Comment (IsraaSulaiman) [inline client/src/App.js]: Why the curly braces? It is a string @aminalakhsham

File: client/src/App.js [hunk 2]
```diff
@@ -1,12 +1,56 @@
...
+              <Route exact path={"/projects"} component={() => <Projects />} />
```
Comment (IsraaSulaiman) [inline client/src/App.js]: Why arrow function here? 
It is better like this
`component={Project}`

Comment (AymanAlqoqa) [inline client/src/App.js]: you need not make it arrow function
just use <Projects />

File: client/src/App.js [hunk 3]
```diff
@@ -1,12 +1,56 @@
...
+              <Route
+                exact
+                path={"/projects/new"}
+                component={() => <ProjectsNew />}
```
Comment (AymanAlqoqa) [inline client/src/App.js]: the same as previous

File: client/src/App.js [hunk 4]
```diff
@@ -1,12 +1,56 @@
...
+                exact
+                path={"/projects/:projectId/edit"}
+                component={ProjectsEdit}
+              />
+              <Route
+                exact
+                path={"/projects/:projectId/:scrumId"}
+                component={Scrum}
+              />
+              <Route exact path={"/members"} component={() => <Members />} />
```
Comment (AymanAlqoqa) [inline client/src/App.js]: you need to add default route for page not found

File: client/src/App.js (hunk 3 above)
Comment (AymanAlqoqa) [reply client/src/App.js]: ok

File: client/src/App.js (hunk 4 above)
Comment (AymanAlqoqa) [reply client/src/App.js]: ok

**Pull request merged**